pkgs/by-name/fa/fastcap/package.nix

```
@@ -0,0 +1,90 @@
+{ stdenv
+, fetchzip
+, ghostscript
+, lib
+, texliveMedium
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fastcap";
+  version = "2.0-18Sep92";
+
+  src = fetchzip {
+    url = "http://www.rle.mit.edu/cpg/codes/fastcap-${version}.tgz";
+    hash = "sha256-fnmC6WNd7xk8fphxkMZUq2+Qz+2mWIP2lvBUBAmUvHI";
+    stripRoot = false;
+  };
+
+  patches = [
+    ./patches/fastcap-mulglobal-drop-conflicting-lib.patch
+    ./patches/fastcap-mulsetup-add-forward-declarations.patch
```

```suggestion
    ./fastcap-mulglobal-drop-conflicting-lib.patch
    ./fastcap-mulsetup-add-forward-declarations.patch
```

_Only a few packages_ do this thing of dedicating a subdirectory for patches. An uniform layout is better achieved by keeping all files in a same directory.

---

OK